src/models/jwtPermissions/index.js

```diff
@@ -132,10 +132,10 @@ const checkAtLeastOnePermissions = (tokenPermissions, permissions) => {
  * @param {number|number[]} permissions
  * @returns {(function(*, *): Promise<void>)|*}
  */
-exports.checkAtLeastOnePermissionsMiddleWare = (permissions) => async (ctx, next) => {
+exports.checkAtLeastOnePermissionMiddleWare = (permissions) => async (ctx, next) => {
     permissions = typeof permissions !== 'object' ? [permissions] : permissions;
     const tokenPermissions = getScope(ctx.state.jwtPayload, PERMISSION_SCOPE)[1];
```

Comment:
if there is no matching scope, an unexpected error will be thrown ("TypeError: Cannot read properties of null")

Reply:
done